packages/stack/src/TransitionConfigs/CardStyleInterpolators.tsx

```diff
@@ -63,6 +63,13 @@ export function forHorizontalIOS({
   };
 }
 
+export function forHorizontalIOSInverted(
+  props: StackCardInterpolationProps
+): StackCardInterpolatedStyle {
+  props.inverted = new Animated.Value(-1);
+  return forHorizontalIOS(props);
```

Comment:
Hardcoding `inverted` to `new Animated.Value(-1)` means any `inverted` property passed here will be ignored. This needs to respect the existing `inverted` value:

```suggestion
 { inverted, ...rest }: StackCardInterpolationProps
): StackCardInterpolatedStyle {
  return forHorizontalIOS({
    ...rest,
    inverted: Animated.multiply(inverted, -1),
  });
```